### DISCUSSION TITLE
Week 5 Summary - Javascript Review Sessions and Froggy Thoughts

### DISCUSSION THREAD

**Discussion by bst3ck** [USU-ITLS-x265-23-summer/class-repo#116] [General]
## Week 5 Summary

This week I decided that I wanted to spend more time in the Javascript tutorial since I have a pretty good understanding of the HTML basics and the css basic and I have a feeling that any of the projects I choose I am going to need some heavy javascript to display the data needed for the website.

I did go back through a couple of the flexbox items in the Froggy tutorial, but that is because I was trying to work out some layout options for what I was thinking about for my final project.

What I found during my javascript tutorial is that I had easily forgotten a lot of the items I had learned a few weeks ago. In order to proceed I had to step a lot further back in the lessons until I was "warmed up". It seems that once I was warmed up, I was able to move more quickly through the lessons and I was able to start to see some of the ways I can use the information in my future project. I think I may need to go through this a few more times to start to get really comfortable with the topics. I am finding I am still spending a bunch of time in google looking up options on how to tackle specific scenarios. I hope to start reduce the amount of time searching for solutions and instead start looking at how to apply these more easily in future activities.

**Comment by metasean** [comment]
I suggest that, as you identify what helps you "warm up" the most, that you add corresponding notes to the class-repo for future reference.  As you run across things that you've learned, but have forgotten, it's a good sign that you'll find it useful to have a quick reference for future use.
